src/plugins/timeConductor/ConductorTimeSystem.vue

```diff
@@ -60,7 +60,9 @@ export default {
                 .filter(menuOption => menuOption.clock === (clock && clock.key))
                 .map(menuOption => JSON.parse(JSON.stringify(this.openmct.time.timeSystems.get(menuOption.timeSystem))));
         },
-
+        bothTimeSystemsUtcBased(timeSystem1, timeSystem2) {
```

Comment:
I would change the function name to something like `isTimeSystemsUtcBased()` to better describe what the function does.

Reply:
Inline the check?  it's concise & clear:

```javascript
if (this.bothTimeSystemsUtcBased(this.selectedTimeSystem, timeSystem)) { /*...*/ }
// vs
if (this.selectedTimeSystem.isUTCBased && timeSystem.isUTCBased) { /*...*/ }
```

Reply:
Yes, that's even better.